### PR TITLE
Change auhtorize account signature

### PR DESCRIPTION
**Pull request by mpnowacki-reef**

### CODE REVIEW

File: changelog.d/+b2cli958.changed.md
Comment (kkalinowski-reef) [inline changelog.d/+b2cli958.changed.md:1]: Isn't it a valid case of having a GitHub ticket, just from a different repo, assigned here?

File: test/unit/v_all/test_api.py [hunk 1]
```diff
@@ -116,7 +116,11 @@ def setUp(self):
         (self.application_key_id, self.master_key) = self.raw_api.create_account()
 
     def _authorize_account(self):
-        self.api.authorize_account('production', self.application_key_id, self.master_key)
+        self.api.authorize_account(
+            realm='production',
```
Comment (kkalinowski-reef) [inline test/unit/v_all/test_api.py:120]: Nitpick: all other changes to `kwargs` resulted in placing `realm` being placed as last in the invocation.

File: test/unit/v_all/test_replication.py [hunk 2]
```diff
@@ -39,7 +39,11 @@ def setUp(self):
         self.application_key_id, self.master_key = self.raw_api.create_account()
 
     def _authorize_account(self):
-        self.api.authorize_account('production', self.application_key_id, self.master_key)
+        self.api.authorize_account(
+            realm='production',
```
Comment (kkalinowski-reef) [inline test/unit/v_all/test_replication.py:43]: Nitpick: all other changes to `kwargs` resulted in placing `realm` being placed as last in the invocation.

File: b2sdk/api.py [hunk 3]
```diff
@@ -209,13 +209,13 @@ def authorize_automatically(self):
         return self.session.authorize_automatically()
 
     @limit_trace_arguments(only=('self', 'realm'))
-    def authorize_account(self, realm, application_key_id, application_key):
+    def authorize_account(self, application_key_id, application_key, realm='production'):
```
Comment (kkalinowski-reef) [inline b2sdk/api.py:212]: I was considering asking for providing remaining `realm` options as e.g. `Literal` type or in the `docstring`. But these additional options are only for B2 developers, right? So it probably wouldn't make sense to add it here :/

**Review by kkalinowski-reef — APPROVED** [review]

File: changelog.d/+b2cli958.changed.md
Comment (mpnowacki-reef) [reply changelog.d/+b2cli958.changed.md:1]: would be, but town crier doesn't support it, so this will just yield a broken link

Comment (kkalinowski-reef) [reply changelog.d/+b2cli958.changed.md:1]: Also, this wouldn't be valid with the latest `nox -s towncrier_check`

**Pull request closed without merging**